packages/builder/src/components/design/settings/controls/PropertyControl.svelte

```diff
@@ -32,7 +31,7 @@
   $: safeValue = getSafeValue(value, defaultValue, allBindings)
   $: replaceBindings = val => readableToRuntimeBinding(allBindings, val)
 
-  $: if (!Array.isArray(value)) {
```

Comment:
This basically disallowed us from highlighting array values, say like an onClick handler. I changed it to this and haven't been able to find any issues arising.